### PR TITLE
Add CI pipeline and MIT license

### PR DESCRIPTION
**Pull request by ddugovic**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,115 @@
...
+
+# node-waf configuration
+.lock-wscript
+
+# Compiled binary addons (https://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directories
 node_modules/
-data/
```
Comment (ebemunk) [inline .gitignore:2]: you should leave these in, i think they were needed

**Review by ebemunk — APPROVED** [review]

**Comment by ebemunk** [comment]
cool! thanks for doing this. i havent touched this repo in a long time but I don't think any of these changes can hurt. please add the 2 deleted items back into `.gitignore` though because I think they were needed for build

**Review by ebemunk — CHANGES_REQUESTED** [review]
add lines back to `.gitignore`

**Comment by ddugovic** [comment]
Oops, sorry about that, and yes those are valuable so I don't accidentally upload local data.  Here you go!

**Review by ebemunk — APPROVED** [review]

**Comment by ebemunk** [comment]
Thanks @ddugovic !

**Pull request merged**